scripts/release/release_info.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import json
+import pathlib
+from typing import Dict
+
+from scripts.release.version import (
+    Environment,
+)
+
+
+class ImageInfo(dict):
+    def __init__(self, repository: str, platforms: list[str], version: str):
+        super().__init__()
+        self.repository = repository
+        self.platforms = platforms
+        self.version = version
+
+    def to_json(self):
+        return {"repository": self.repository, "platforms": self.platforms, "version": self.version}
+
+
+class BinaryInfo(dict):
+    def __init__(self, s3_store: str, platforms: list[str], version: str):
+        super().__init__()
+        self.s3_store = s3_store
+        self.platforms = platforms
+        self.version = version
+
+    def to_json(self):
+        return {"platforms": self.platforms, "version": self.version}
+
```

Comment:
Did we leave `s3_store` intentionally from returned json?

Reply:
yes, `to_json` is only used when generating `release_info.json` that is used by customers. `s3_store` has no purpose for them, nor we should expose this internal information